runtime/python_bridge.py

```
@@ -326,55 +395,185 @@ def write_payload(payload: str) -> bool:
+    frames = encode_frames(
+        payload,
+        id=response_id,
+        stream='response',
+        max_frame_bytes=MAX_FRAME_BYTES,
+    )
+    for frame in frames:
+        # One frame per JSONL line; flush per frame so the pipe backpressures
+        # and the TS reader can interleave reassembly with the write.
+        if not write_payload(_response_codec.encode(frame)):
+            return False
+    return True
```

Ruff (0.15.15)

[warning] Use `return all(write_payload(_response_codec.encode(frame)) for frame in frames)` instead of `for` loop

Replace with `return all(write_payload(_response_codec.encode(frame)) for frame in frames)`

(SIM110)